docs/source/tutorials/static-analysis.md

```diff
@@ -0,0 +1,286 @@
+---
```

Comment:
I think you also need change sidebar and em.yml, to make this file visible on sidebar.